src/utils/seedRefHooks.ts

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+
+import { SeedRef } from 'utils/native';
+
+type IsValidFunc = () => boolean;
+type TryCreateFunc = (encryptedSeed: string, password: string) => Promise<void>;
+type TryDestroyFunc = () => Promise<void>;
+type TrySignFunc = (message: string) => Promise<string>;
+
+type SeedRefHooks = [IsValidFunc, TryCreateFunc, TryDestroyFunc, TrySignFunc];
```

Comment:
BTW here, what about changing them to a object/enum, because now to import sign function we need do 
```
const [isvalid, , , , substrateSign] = useSeedRef();
if(isvalid) {
 substrateSign(message)
}
```

with enum could be 
```
const [isValid, {substrateSign}] = useSeedRef();
if(isvalid) {
 substrateSign(message)
}
```

which could be more clear, BTW with the latter one you have to specify the types of each function, I just tried yesterdat...